### PR TITLE
Fix dropdown item hover bgcolor in dark mode

### PR DESCRIPTION
**Pull request by allcaps**
# Before

Screenshot 2022-11-05 at 02 08 19

# After

Screenshot 2022-11-05 at 02 08 40

### CODE REVIEW

File: apps/frontend/static_src/scss/components/language-selector.scss [hunk 1]
```diff
@@ -49,5 +49,11 @@
             background-color: $color--black;
             color: $color--light-grey;
         }
```
Comment (nicklee) [inline apps/frontend/static_src/scss/components/language-selector.scss:51]: Small nitpick: suprised the linter didn't suggest having a space between these two?

**Comment by nicklee** [comment]
LGTM, nice one!

**Comment by Scotchester** [comment]
How did the alignment of the dropdown change from aligned on the right side of the button to aligned on the left side of the button? 🤔

**Comment by allcaps** [comment]
> How did the alignment of the dropdown change from aligned on the right side of the button to aligned on the left side of the button? 🤔

I don't know, must be something during the screenshot, as the Gitpod and the live site behave the same.

**Pull request merged**